crates/rooch/src/commands/move_cli/commands/integration_test.rs

```diff
@@ -238,8 +246,8 @@ impl IntegrationTest {
             test_args.push(filter);
         }
 
-        let test_opts = datatest_stable::TestOpts::try_parse_from(test_args.as_slice())?;
```

Comment:
The `datatest_stable::TestOpt` can support filters, for example, running a specific test or running tests via prefix.

It also can run tests in multi-thread,  I think the datatest_stable is useful, we can refactor it.

Reply:
The issue has been fixed, and we have resumed using datatest for testing.